tutorials/detecting-changes-in-sentinel-1-imagery-pt-1/tutorial.ipynb

```diff
@@ -0,0 +1 @@
age is\n","\n","$$\n","\\langle s\\rangle = {1\\over m}\\sum_{i=1}^m s_i \\tag{1.17}\n","$$\n","\n","and its mean value is\n","\n","$$\n","{\\rm mean}(\\langle s\\rangle) = {1\\over m}\\sum_{i=1}^m {\\rm mean}(s_i) = {1\\over m}\\sum_{i=1}^m a = a. \\tag{1.18}\n","$$\n","\n","Now we see that the histogram of the Sentinel-1 multi-look image $\\langle s\\rangle =\\langle |S_{vv}|^2 \\rangle$ follows a gamma distribution with the parameters \n","\n","$$\n","\\alpha=m,\\quad \\beta' = {a\\over m} = {{\\rm mean}(\\langle s\\rangle)\\over m}, \\tag{1.19}\n","$$\n","\n","as we demonstrated earlier with the measured histogram.\n","\n","The covariance representation of the dual pol multilook images is \n","\n","$$\n","C2 = \\pmatrix{\\langle|S_{vv}|^2\\rangle & 0 \\cr 0 & \\langle|S_{vh}|^2\\rangle}. \\tag{1.20}\n","$$\n","\n","\n","### Equivalent number of looks\n","\n","The variance of $\\langle s\\rangle$ is given by\n","\n","$$\n","{\\rm var}(\\langle s\\rangle) = {1\\over m^2}{\\rm var}(\\sum_{i=1}^m s_i)= {1\\over m^2}ma^2 = {a^2\\over m}, \\tag{1.21}\n","$$\n","\n","where we have used the fact that the variance of the gamma distribution is $\\alpha\\beta'^2=ma^2$. Thus the variance of the look-averaged image, the speckle effect, decreases inversely with the number of looks.\n","\n","\n","In practice, the neighborhood pixel intensities contributing to the look average will not be completely independent, but correlated to some extent. This is accounted for by defining an _equivalent number of looks_\n","(ENL) whose definition is motivated by Eq. (1.21), that is,\n","\n","$$\n","{\\rm ENL} = {a^2\\over {\\rm var}(\\langle s\\rangle)} = {{\\rm mean}(\\langle s\\rangle)^2\\over {\\rm var}(\\langle s\\rangle)}.\\tag{1.22}\n","$$\n","\n","In general it will be smaller than $m$. Let's see what we get for our subset of the airport image:"]},{"cell_type":"code","metadata":{"id":"3FIc22YZbI95","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"executionInfo":{"status":"ok","timestamp":1598689540871,"user_tz":-120,"elapsed":783,"user":{"displayName":"Mort Canty","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14Gic8wrtuzbljC46e5I9nhzEOgqYcVXCDqSxyrEQ1Q=s64","userId":"11792105515663483880"}},"outputId":"0f585cf5-d129-4583-8e41-d006a38703a9"},"source":["mean**2/variance"],"execution_count":14,"outputs":[{"output_type":"execute_result","data":{"text/plain":["4.324058395540761"]},"metadata":{"tags":[]},"execution_count":14}]},{"cell_type":"markdown","metadata":{"id":"hMrNjW56bNXF","colab_type":"text"},"source":["The value given by the provider (ESA) for the IW mode imagery in the GEE archive is ENL = 4.4, an average over all swaths, so our spatial subset seems to be fairly representative.\n","\n","### Outlook\n","\n","Now we have a good idea of the statistics of the Sentinel-1 images on the GEE. In Part 2 of the Tutorial we will discuss statistical methods to detect changes in two Sentinel-1 images acquired at different times."]}]}
```

Comment:
<ul><li>Clear the output for this section - additionally, I recommend clearing outputs for all sections in case they don't render consistently across platforms (Colab, GitHub, DevSite, etc).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/214/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done - verified.